### PR TITLE
fix #39525 roseline.club

### LINKED ISSUE #39525: roseline.club
### Issue URL (Ads)
[NSFW][http://www.roseline.club/27/buyuk-yarakli-adamla-sikismek-icin-ozenle-hazirlaniyor-brazzers-izle/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.roseline.club%2F27%2Fbuyuk-yarakli-adamla-sikismek-icin-ozenle-hazirlaniyor-brazzers-izle%2F)
### Comment
Username: @VivaLaVidaValeVoua
> 
> --
> Aşağıdaki bilgiler otomatik olarak oluşturuldu. Silmeyin.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js
> https://adsbypasser.github.io/releases/adsbypasser.full.es5.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.2.2903.0
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Self-destructing third-party cookies (0), Block WebRTC, Block Push API, Block Location API
Filters: | EasyList,<br>AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard Base,<br>Adblock Warning Removal List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/39525

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -80,6 +80,9 @@ dizifilmix.org,sineturk.pw,altyazilifilmizle.tv,1080pfilmizle.net,dolufilmizle.c
 fullkurulum.com##center > a[href^="http://bc.vc/"] > img
 ||loginhere.info/24agustos.js
 indirful.com##.code-block > p > strong > a[href][rel="nofollow noopener"]
+roseline.club##div[class^="rosel-icerikten"]
+roseline.club##.toggler
+://www.roseline.club/porno.gif
```
Comment (BlazDT) [inline TurkishFilter/sections/specific.txt]: porno.gif$domain=roseline.club

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -80,6 +80,9 @@ dizifilmix.org,sineturk.pw,altyazilifilmizle.tv,1080pfilmizle.net,dolufilmizle.c
 fullkurulum.com##center > a[href^="http://bc.vc/"] > img
 ||loginhere.info/24agustos.js
 indirful.com##.code-block > p > strong > a[href][rel="nofollow noopener"]
+roseline.club##div[class^="rosel-icerikten"]
+roseline.club##.toggler
```
Comment (BlazDT) [inline TurkishFilter/sections/specific.txt]: Not an ad, expands footer.

File: TurkishFilter/sections/general_extensions.txt [hunk 3]
```diff
@@ -27,6 +27,9 @@ fullhdfilmizlesene.org,ultrahdfilmizle.org#%#AG_onLoad(function() { window.start
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39485
 hintfilmcenneti.pw#%#(function(){var c=window.setInterval;window.setInterval=function(f,g){return g===1e3&&/_0x/.test(f.toString())&&(g*=0.01),c.apply(this,arguments)}.bind(window)})();
+! https://github.com/AdguardTeam/AdguardFilters/issues/39525
+roseline.club#%#//scriptlet("prevent-window-open")
+roseline.club#%#window.open = function() { };
```
Comment (BlazDT) [inline TurkishFilter/sections/general_extensions.txt]: Both rules can be removed, this one is enough when I try. Please also add the domain to "puShown" rule ( AG_setConstant('puShown', 'true'); ).
||roseline.club/dmn.php

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
Exchanged the popup rule and added similar site https://github.com/AdguardTeam/AdguardFilters/issues/39529 to this pr.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**